### PR TITLE
brand news image placeholder

### PR DESCRIPTION
**Pull request by ecotopian**
https://ecosia.atlassian.net/browse/MOB-1471

### CODE REVIEW

**Review by jman82 — APPROVED** [review]
Looks awesome! :rocket:

**Review by vauxhall — APPROVED** [review]
🎨

File: Client/Ecosia/UI/SemanticColor.swift [hunk 1]
```diff
@@ -77,6 +77,10 @@ extension UIColor {
             static let primary = UIColor.white
         }
         
-        static let border = UIColor(red: 0.35, green: 0.35, blue: 0.35, alpha: 1)
+        static let border = Grey.fifty
+    }
+
+    struct Grey {
+        static let fifty = UIColor(red: 0.35, green: 0.35, blue: 0.35, alpha: 1)
```
[the remark this answers is not in the document]
Comment (ecotopian) [reply Client/Ecosia/UI/SemanticColor.swift:84]: it's defined here: https://www.figma.com/file/gEDmMnMcQQxvnpeVbtQ5Z9/iOS-Master?node-id=1167%3A54412

[the remark this answers is not in the document]
Comment (vauxhall) [reply Client/Ecosia/UI/SemanticColor.swift:84]: Better in synch than inventing new names 👍

**Pull request merged**